deepchecks/checks/performance/calibration_score.py

```diff
@@ -52,7 +52,7 @@ def _calibration_score(self, dataset: Dataset, model):
         briers_scores = {}
 
         if len(dataset.classes) == 2:
-            briers_scores[0] = brier_score_loss(ds_y, y_pred[:, 1])
+            briers_scores[0] = brier_score_loss(ds_y, y_pred[:, 1], pos_label=dataset.classes[1])
```

Comment:
Still needed if the results are passed as [0,1]?

Reply:
no but it doesn't matter